CMakeLists.txt

```diff
@@ -0,0 +1,245 @@
+cmake_minimum_required(VERSION 3.16)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+
+# See release_checklist.md
+set(MAJOR_VERSION 2)
+set(MINOR_VERSION 1)
+set(MICRO_VERSION 0)
+set(SDL_REQUIRED_VERSION 2.0.0)
+
+set(DYLIB_COMPATIBILITY_VERSION "1")
```

Comment:
Can you please update `test-versioning.sh` to include `CMakeLists.txt`?

Reply:
Added in fd6bc05520904e64d910b810bf1dbd2e330b008f